### ISSUE TITLE
[Request] Add Spawn Config

### ISSUE THREAD

**Issue by katubug** [EterDelta/CrittersAndCompanions#36]
Hi there! So I want to preface this by saying I absolutely adore this mod (and all your other ones). Thank you so much for making them!

However, we have been having some issues with certain creatures either overspawning (dragonflies) or underspawning (red pandas), and I'd love to be able to tweak how they spawn. I've actually been using a mod, Spawn Balance Utility, to try and do this, but it seems that the creatures in this mod don't obey those spawn rules, as I disabled Dragonflies but they're still spawning.

Help would be much appreciated! Thanks for your time!

**Comment by Vide0Master** [comment]
> overspawning

i will preffer word "**swarming**"

<img width="2560" height="1380" alt="Image" src="https://github.com/user-attachments/assets/0c400ca4-e5f6-47af-8c4b-3aa6af408270" />

<img width="2560" height="1380" alt="Image" src="https://github.com/user-attachments/assets/651d7b41-9960-4687-97ae-1f9b53da9d4f" />

i dont understand why this happenning

**Comment by YumatanGames** [comment]
Yeah, I came here to open a ticket exactly for this reason. Dragonflies are spawning way too frequently. While I have yet to see a single Red Panda in my world. At least they seem to be very well optimized, as server performance doesn't seem to be taking a hit. But still, visually it is not good...
